index.js

```diff
@@ -0,0 +1,31 @@
+const serverless = require("serverless-http");
```

Comment:
This file should be within `serverless/`

Reply:
Serverless folder should've been removed but the changes didn't get pushed, can you check again please?